lib/Raven/Breadcrumbs/ErrorHandler.php

```diff
@@ -41,7 +41,11 @@ public function handleError($code, $message, $file = '', $line = 0, $context = a
 
     public function install()
     {
-        $this->existingHandler = set_error_handler(array($this, 'handleError'), E_ALL);
+        $this->existingHandler = set_error_handler(
```

Comment:
> Why did you change this? 

Because we use PHP 5.5+ not a PHP 4 or something old shit like that

> we have one more function call for nothing

So what? 
https://3v4l.org/2RuH4 9 \* 10 ^ -8 second per a function call. 

If you want to save nanoseconds per function calls, just choose some other language which supports JIT. I.e. C# or Java or PHP 8